presto-hive/src/main/java/com/facebook/presto/hive/statistics/MetastoreHiveStatisticsProvider.java

```diff
@@ -102,6 +175,67 @@ public TableStatistics getTableStatistics(ConnectorSession session, ConnectorTab
         return tableStatistics.build();
     }
 
+    private boolean isLowHighSupportedForType(Type type)
+    {
+        if (type instanceof DecimalType) {
+            return true;
+        }
+        if (type.equals(TINYINT)
+                || type.equals(SMALLINT)
+                || type.equals(INTEGER)
+                || type.equals(BIGINT)
+                || type.equals(REAL)
+                || type.equals(DOUBLE)
+                || type.equals(DATE)
+                || type.equals(TIMESTAMP)) {
+            return true;
+        }
+        return false;
+    }
+
+    private Object highLowValueAsPrestoType(Object value, Type prestoType)
```

Comment:
call this `lowHighValueAsPrestoType` so that it matches `isLowHighSupportedForType`